citation-backoffice/src/components/forms/Object.js

```diff
@@ -27,22 +27,23 @@ class GenericObject extends Component {
 		Object.keys(result[type]).forEach(key => {
```

Comment:
Still convinced that this forEach should be refacto in something like `return _.map(result[type])` but I think not in this PR...